### ISSUE TITLE
Speed on Titan X

### ISSUE THREAD

**Issue by athindran** [thtrieu/darkflow#425]
Hi,

  What speed can I expect when I run the darkflow YOLOv2 detector with input size 416x416 on Titan X? For me, it is only running at ~30 fps. Is it not supposed to run at 67 fps? Is this normal or am I missing something? I am using yolo-voc.cfg and yolo-voc.weights.

**Comment by rlan** [comment]
Are you quoting fps from Redmon's paper? I don't know how fast darkflow can be, but the code base for his paper is in darknet.

**Comment by athindran** [comment]
Yes. I am quoting redmon's paper? Is darkflow slower? Anybody has any idea?

**Comment by lzane** [comment]
I run `--demo` on a video and got around 55fps on Titan X and around 8fps on TX2. Anybody try tiny yolo using darknet? please tell me their performance

**Comment by OseongKwon** [comment]
@Izane I've aquired the 17fps with tiny-yolo-voc at TX2.

**Comment by dht7166** [comment]
Can you be more specific how you use achieve 17fps?

**Comment by rummanwaqar** [comment]
@lzane did you run it using TensorRT on TX2 or directly?

**Comment by lzane** [comment]
@rummanwaqar I run it directly.

**Comment by lzane** [comment]
@rummanwaqar If you use darknet, the c++ version implementation, you can get a better result at around 20fps

**Comment by rummanwaqar** [comment]
@lzane I am running directly c++ implementation with yolov2 network on TX2 and I get around ~5-6 fps

**Comment by dht7166** [comment]
Apparently if you used a frozen model, it might have a slight speed boost. I tried on TX2, so it might differ quite a bit.

**Comment by rummanwaqar** [comment]
@dht7166 whats a frozen model?

**Comment by pabloapast** [comment]
Hey @lzane @OseongKwon, can you give more details on how did you get souch higher fps rate on the tx2? did you use the original Redmon's Darknet implementation in C or did you use a version written in C++ which is faster?
I've found [this paper](http://openaccess.thecvf.com/content_ICCV_2017_workshops/papers/w30/Tijtgat_Embedded_Real-Time_Object_ICCV_2017_paper.pdf) where the authors only got ~11fps using tiny-yolo on tx2.

**Comment by lzane** [comment]
@pabloapast I use this version of [darknet](https://github.com/AlexeyAB/darknet).

Remember to set tx2  to the high performance mode.
